lib/strong_resources/controller/mixin.rb

```diff
@@ -8,6 +8,11 @@ class << self
             attr_accessor :_strong_resources
           end
         end
+
+        if respond_to?(:register_exception)
```

Comment:
maybe a quick comment on the respond to, nothing that we're integrating with errorable